metricflow-semantics/metricflow_semantics/query/validation_rules/metric_time_requirements.py

```diff
@@ -192,19 +189,4 @@ def validate_measure_in_resolution_dag(
         measure_reference: MeasureReference,
         resolution_path: MetricFlowQueryResolutionPath,
     ) -> MetricFlowQueryResolutionIssueSet:
-        scd_linkable_elemenent_set_for_measure = self._scd_linkable_element_set_for_measure(measure_reference)
```

Comment:
Seems like we can now remove `_scd_linkable_element_set_for_measure` and the mention of SCD in the class docstring?